code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -2033,7 +2040,7 @@ datum
 			name = "Moonshine"
 			id = "moonshine"
 			result = "moonshine"
-			required_reagents = list("nutriment" = 10)
+			required_reagents = list("cornoil" = 10)
```

Comment:
In addition, most every other alchohol in this file requires something from botany.